internal/profile/certs.go

```diff
@@ -0,0 +1,160 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package profile
+
+import (
+	"bytes"
+	"crypto/x509"
+	"fmt"
+	"io"
+	"path/filepath"
+
+	"github.com/elastic/elastic-package/internal/certs"
+)
+
+// tlsServices is the list of server TLS certificates that will be
+// created in the given path.
+var tlsServices = []string{
+	"elasticsearch",
+	"kibana",
+	"package-registry",
+	"fleet-server",
+}
+
+var (
+	// CertificatesDirectory is the path to the certificates directory inside a profile.
+	CertificatesDirectory = "certs"
+
+	// CACertificateFile is the path to the CA certificate file inside a profile.
+	CACertificateFile = configFile(filepath.Join(CertificatesDirectory, "ca-cert.pem"))
+
+	// CAKeyFile is the path to the CA key file inside a profile.
+	CAKeyFile = configFile(filepath.Join(CertificatesDirectory, "ca-key.pem"))
```

Comment:
nit: should these values be exposed?

Reply:
I left them as other profile files that can be later requested with `profile.FetchPath(...)`. Though only `CACertificateFile` is used now this way, so I could make the others private, as you prefer.

Reply:
Got it, it's consistent with other variables. Let's leave it as is.